content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md

```diff
@@ -195,4 +195,27 @@ openssl command is expected.
     $ kubectl delete ns foo istio-system
     {{< /text >}}
 
-*   To remove the Istio components: follow the [uninstall instructions](/docs/setup/getting-started/#uninstall) to remove.
+*   Remove the authentication policy for the foo namespace:
```

Comment:
```suggestion
*   Remove the authentication policy from the foo namespace:
```

Reply:
`for` is equally correct in this case because it is "the policy for the foo namespace" that's being delete (from the foo namespace).